examples/gke-tpu-v4-xpk/gke-tpu-v4-deployment.yaml

```diff
@@ -0,0 +1,53 @@
+# Copyright 2025 "Google LLC"
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+terraform_backend_defaults:
+  type: gcs
+  configuration:
+    # The GCS bucket used for storing terraform state
+    bucket: tas-terraform
+
+vars:
+  # Your GCP Project ID
+  project_id: cloud-tpu-multipod-dev
+
+  # This should be unique across all of your Cluster
+  # Toolkit Deployments.
+  deployment_name: tpu-v4-swarnabm3 # gke-tpu-v4
+
+  # The GCP Region used for this deployment.
+  region: us-central2
+
+  # The GCP Zone used for this deployment.
+  zone: us-central2-b
+
+  # The number of nodepools / TPU slices to create
+  num_node_pools: 2
+
+  # The TPU placement topology for pod slice node pool.
+  tpu_topology: 2x2x4
+
+  # The number of nodes to be created in each nodepool
+  # For TPUs, node_count should be num of chips based on topology / num chips for the machine type
+  # For v4, if topology is 2x2x4, then node count will be 16/4 = 4
+  static_node_count: 4
+
+  # Cidr block containing the IP of the machine calling terraform.
+  # To allow all (IAM restrictions still enforced), use 0.0.0.0/0
+  # To allow only your IP address, use <YOUR-IP-ADDRESS>/32
+  authorized_cidr: 0.0.0.0/0
+
+  # The name of the compute engine reservation of A4-High nodes in the form of
+  # <project>/<reservation-name>/reservationBlocks/<reservation-block-name>
+  extended_reservation: cloudtpu-us-central2-b-v4-pod-735972712744
```

Comment:
here too, well, basically the whole deployment file should not have identifiable values ;)